tutorial/Lab3/README.md

```diff
@@ -0,0 +1,84 @@
+2. Roll back to the previous revision:
+
+    In this rollback, Helm checks the changes that occured when upgrading from the revision 1 to revision 2. This information enables it to makes the calls to the Kubernetes API server, to update the deployed application as per the initial deployment - in other words with Redis slaves and using a load balancer.
+
+    Rollback with this command, ```$ helm rollback guestbook-demo 1```
+    
+    ```console
+    Rollback was a success! Happy Helming!
+    ```
+    Check the history again, `helm history guestbook-demo`
+    
+    You should see output similar to the following:
+    
+    ```console
+    REVISION	UPDATED                 	STATUS    	CHART          	DESCRIPTION     
+    1       	Mon Sep 24 08:54:04 2018	SUPERSEDED	guestbook-0.1.0	Install complete
+    2       	Mon Sep 24 10:36:18 2018	SUPERSEDED	guestbook-0.1.0	Upgrade complete
+    3       	Mon Sep 24 11:59:18 2018	DEPLOYED  	guestbook-0.1.0	Rollback to 1
+    ```
+    
+    To check the rollback, you can run `kubectl get all --namespace helm-demo`:
+    
+    ```console
+    NAME                                READY     STATUS    RESTARTS   AGE
+    pod/guestbook-demo-6c9cf8b9-dhqk9   1/1       Running   0          3h
+    pod/guestbook-demo-6c9cf8b9-zddn2   1/1       Running   0          3h
+    pod/redis-master-5d8b66464f-g7sh6   1/1       Running   0          3h
+    pod/redis-slave-586b4c847c-tkfj5    1/1       Running   0          2m
+    pod/redis-slave-586b4c847c-xxrdn    1/1       Running   0          2m
+    
+    NAME                     TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+    service/guestbook-demo   LoadBalancer   172.21.43.244   <pending>     3000:31202/TCP   3h
+    service/redis-master     ClusterIP      172.21.12.43    <none>        6379/TCP         3h
+    service/redis-slave      ClusterIP      172.21.232.16   <none>        6379/TCP         2m
+    
+    NAME                             DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+    deployment.apps/guestbook-demo   2         2         2            2           3h
+    deployment.apps/redis-master     1         1         1            1           3h
+    deployment.apps/redis-slave      2         2         2            2           2m
+    
+    NAME                                      DESIRED   CURRENT   READY     AGE
+    replicaset.apps/guestbook-demo-6c9cf8b9   2         2         2         3h
+    replicaset.apps/redis-master-5d8b66464f   1         1         1         3h
+    replicaset.apps/redis-slave-586b4c847c    2         2         2         2m
+    ```
+   
+    You can see from the output that the app service is the service type of `LoadBalancer` again and the Redis master/slave deployment has returned. 
+    This shows a complete rollback from the upgrade in [Lab 2](../Lab2/README.md) 
+
+# Conclusion
+
+From this lab, we can say that Helm does revision management well and Kubernetes does not have the capability built in!
```

Comment:
I wonder if it would make sense to distinguish between this formal rollback mechanism and simply doing another "helm install" with a different set of values? The net result is the same but the rollback doesn't require the user to remember all of the various values. Are there other benefits or differences?

Reply:
I would think they are 2 separate things - install of new instance and management of an existing instance. I am not sure what value this would add to the lab and the reader.